### DISCUSSION TITLE
Request for new standard name: land_ice_transport_across_line

### DISCUSSION THREAD

**Discussion by mankoff** [cf-convention/discuss#388] [Comments and ideas for changing CF]
### Topic for discussion

Name: `land_ice_transport_across_line`
Units: `kg s-1`

Description: Similar to `sea_water_transport_across_line`: Transport across_line means that which crosses a particular line on the Earth's surface; formally this means the integral along the line of the normal component of the transport.

That is, we need a variable for iceberg discharge in Greenland and Antarctica. `water_flux_into_sea_water_from_{land_ice,icebergs}` is close, but has the wrong units (flux needs `m-2`) and this new term is for solid ice, not water.

**Comment by taylor13** [comment]
In descriptions of land_ice, we say:

> "Land ice" means glaciers, ice-caps and ice-sheets resting on bedrock and also includes ice-shelves.

So I think ``iceberg_transport_across_line`` would be better.  Note that "icebergs" is already used in 3 standard names.  This would, of course, exclude flow of ice_shelves across the line.

**Reply by mankoff** [reply]
Agreed that what I suggested should be `iceberg_transport_across_line`. But then there should also be the term I suggested which has the same units but updated definition: Discharge across the grounding line (or any other line) but remains `land_ice` after crossing said line, unlike icebergs. The use case here is ice entering (creating) ice shelves on the supply side, rather than ice leaving ice shelves on the calving side.

**Reply by mankoff** [reply]
More generally, `ice_transport_across_line` could work in both cases.

**Comment by taylor13** [comment]
Regarding the second (new) case, if the line followed a coast line, I agree that ice_transport_across_line might work (except we might worry about ice-clouds in the atmosphere mistakenly being included), but if the line were out at sea, then "ice transport" would definitely include sea-ice.  If sea ice should be excluded, I think "land_ice_transport ..." would be better in this case, which would include flow from land to sea or flow of ice-shelves when no longer grounded, but it would exclude sea ice and icebergs, I guess.  
Anyone else want to weigh in on this?

**Comment by JonathanGregory** [comment]
I agree that `iceberg_transport_across_line` and `land_ice_transport_across_line` would both be suitable as standard names if they are needed. The "lines" referred to by such quantities are geographically defined, whereas the grounding line is a mobile geophysical boundary. Therefore I thnk `land_ice_transport_across_grouding_line` would also be a reasonable quantity to name.

**Comment by efisher008** [comment]
Hi all,

Could this be converted into an issue in the vocabularies repository? This would be the preferred route as it is how we usually manage proposals for new standard names. Alternatively, a new issue can be opened with the final proposal once the term name and description have been agreed.

Best regards,
Ellie

**Reply by mankoff** [reply]
Hi @efisher008,

I wanted to open this as an issue, per the instructions under "How do I propose a change?" at https://github.com/cf-convention/cf-convention.github.io/blob/main/faq.md#how-do-i-propose-a-change . The text and links there are to new issues at https://github.com/cf-convention/discuss/issues but if you try to open an issue there, it pushes you to this discussion forum.

**Reply by efisher008** [reply]
Hi @mankoff,

I think I see what has happened. The location for submitting CF Standard Name proposals moved in July this year from the cf-convention/discuss repo to the newly created cf-convention/vocabularies repo. This was done to provide a clearly labelled repo which collects together all discussions and resources related to the CF controlled vocabularies (standard names, area types and standardized regions). 

The history of this change can be found here:
https://github.com/orgs/cf-convention/discussions/334

And the new repository for vocabularies is here:
https://github.com/cf-convention/vocabularies

It seems as though the page you found still directs to the old discuss repo, which was not caught when changing the links on the website to point to the new vocabularies repo. I will raise a request for this to be changed!

**Comment by JonathanGregory** [comment]
Sorry, Ken @mankoff. I will update the FAQ. Thanks for discovering this.

**Comment by JonathanGregory** [comment]
I have opened [website issue 553](https://github.com/cf-convention/cf-convention.github.io/issues/553) to update the FAQ.

**Comment by efisher008** [comment]
Hi Ken @mankoff, Jonathan @JonathanGregory,

Putting aside the website issue Jonathan has now created, would it now be possible to open a formal proposal in the [cf-convention/vocabularies](https://github.com/cf-convention/vocabularies) repo (by creating a new issue using the CF Standard Names template, process guidance [here](https://cfconventions.org/standard_name_rules.html)) as it seems this discussion has reached a consensus on what is needed?

If I understood correctly, the proposal would consist of the following names:
`iceberg_transport_across_line`
`land_ice_transport_across_line`
`land_ice_transport_across_grouding_line`
with descriptions and canonical units (which must be physically consistent with a [P06 term](https://vocab.nerc.ac.uk/search_nvs/P06/) in the NERC Vocab Server). 

@mankoff I will leave it up to your discretion how to go forward with the proposal.

Best wishes,
Ellie

**Comment by JonathanGregory** [comment]
I agree with you, Ellie. If Ken @mankoff has time to make the proposal, I think we will agree it easily. A new standard names issue can be opened with [this link](https://github.com/cf-convention/vocabularies/issues/new?assignees=&labels=add+to+cfeditor%2C+standard+name&projects=&template=standard-names.md&title=+Standard+names%3A+%2Aadd+your+own+title+here%2A).

**Comment by mankoff** [comment]
I've opened https://github.com/cf-convention/vocabularies/issues/235

But I'm not sure I should have - reasons included in that ticket. Closing this as this entire discussion should have been there.
